<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLiquidSamplingCheck</class>
 <widget class="QDialog" name="DialogLiquidSamplingCheck">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>取液任务启动确认</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QCheckBox" name="checkBox_power">
    <property name="text">
     <string>请确认系统启动（绿灯）</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_light">
    <property name="text">
     <string>请确认手套箱光照（亮灯）</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_obstacle">
    <property name="text">
     <string>请确认瓶架就位/无障碍物阻挡</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_sink">
    <property name="text">
     <string>请确认取液瓶/废液盒就位</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_purge">
    <property name="text">
     <string>请确认吹扫气压力（3.5~4.5barA）</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_exp_stop">
    <property name="text">
     <string>请确认各收集反应釜处于“试验停止”状态</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_tip">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>即将启动当前任务。。。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLiquidSamplingCheck</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLiquidSamplingCheck</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
